### ISSUE TITLE
Prenotami Project

### ISSUE THREAD

**Issue by VitorCostaMarques** [ValikProgrammer/VisaItaly#1]
Hello Valik,

I am Vitor from Brazil and I saw your project "https://github.com/ValikProgrammer/VisaItaly" which made me think about if it is possible to make an adaptation for other services on the website: https://prenotami.esteri.it/

This site is used to make reservations for some consular services abroad. And since all the public reservation services in the world are always inefficient, I think that an adaptation of your project can be a solution.

I would like to know your availability to do this job.

We can also make a video call to talk about the value of the services and how to proceed.

my whatsapp: +55 21 99120 2793
Email: Vitorcostaeng@gmail.com

regards
Vitor Costa Marques

**Comment by ferksr** [comment]
@VitorCostaMarques I agree with your opinion. Did you find any solution for this? That feature would be requiered by a lot of people, including me (and it can be profitable).
@ValikProgrammer

**Comment by oscar-defelice** [comment]
I am in if you want! I am looking for the passport reservation :D

**Comment by mmartinez85** [comment]
Has anyone been successful in running this script?
